src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/UpgradeAssistantHostExtensions.cs

```diff
@@ -183,13 +187,31 @@ private static void AddUserInput(this IServiceCollection services)
                     services.AddSingleton(parseResult);
                     services.AddTransient<IUpgradeStartup, UsedCommandTelemetry>();
                 })
-                .UseSerilog((_, __, loggerConfiguration) => loggerConfiguration
+                .UseSerilog((context, __, loggerConfiguration) => loggerConfiguration
                     .Enrich.FromLogContext()
-                    .MinimumLevel.Is(Serilog.Events.LogEventLevel.Verbose)
+                    .MinimumLevelFromConfiguration(context.Configuration.GetSection("Logging:Loglevel"))
+                    .MinimumLevel.Override("System", LogEventLevel.Warning)
```

Comment:
Why is this needed?

Reply:
Is this  to handle something along the lines of what I'm seeing here - https://github.com/dotnet/upgrade-assistant/pull/938/checks?check_run_id=4255975097 -  when I changed the .NET SDK version to `6.0.100`?

Reply:
No, that's unrelated.

This is about using HttpClient emits informational stuff about what's going on - we don't need to show that